src/NzbDrone.Core.Test/DecisionEngineTests/HistorySpecificationFixture.cs

```diff
@@ -113,106 +112,106 @@ public void should_return_true_if_latest_history_item_is_not_grabbed()
         [Test]
         public void should_return_true_if_latest_history_item_is_older_than_twelve_hours()
         {
-            GivenMostRecentForEpisode(FIRST_EPISODE_ID, string.Empty, _notupgradableQuality, DateTime.UtcNow.AddHours(-13), HistoryEventType.Grabbed);
+            GivenMostRecentForAlbum(FIRST_ALBUM_ID, string.Empty, _notupgradableQuality, DateTime.UtcNow.AddHours(-13), HistoryEventType.Grabbed);
```

Comment:
Changed to 12 hours 1 millisecond. Not a huge deal, but you are right this is more accurate test.